### DISCUSSION TITLE
How to generate personal access token?

### DISCUSSION THREAD

**Discussion by skorphil** [community/community#70567] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Hi, i'm trying to clone my private repo without success. I have read docs - https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens?ref=hunj-dev

There written I have to generate access toke, but I don't have this feature. I have like gpg and ssh and I cant generate em on that page.
![Screenshot 2023-10-19 at 20 55 59](https://github.com/community/community/assets/6762009/dc0d0dfc-6899-4438-9b00-97d637fa8bab)

How to clone private repo?

**Comment by skorphil** [comment, marked as answer]
Found it in developer options in the very bottom of settings page

**Reply by Jayzroc** [reply]
How do I get my bitcoin
